frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -76,12 +64,14 @@ const Header: React.FC = () => {
           <Link to="/">
             <Logo />
           </Link>
-          <Divider orientation="vertical" flexItem />
-          <Title variant="h5">clutch</Title>
+          <Title>clutch</Title>
           <Box />
           <SearchField />
           <Box />
-          <UserInformation />
+          <Grid container alignItems="center" justify="flex-end">
```

Comment:
since the designs have the search bar, notification icon, and user info grouped to the right, I added this grid component here.